.github/workflows/publish.yml

```diff
@@ -15,8 +14,7 @@ jobs:
     steps:
       # Checks-out the repository under $GITHUB_WORKSPACE, so it's accessible to the job
       - uses: actions/checkout@v2
-
-      - name: Build and publish a collection
-        uses: artis3n/ansible_galaxy_collection@v2
+      - name: galaxy
+        uses: robertdebock/galaxy-action@2da9916b23b62d328af9c8814ad5973d68817a7e
```

Comment:
Seems that this is not a complete solution - I audited the action (as you are recommended to do), and it uses this Docker image: https://hub.docker.com/r/robertdebock/github-action-galaxy. So while the action code might not change, the image could. There is no way to force a specific tag (and they are mutable anyway).

I guess this is nothing new - we are already dependent upon mutable things in our supply chain.